apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -88,13 +112,34 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
         setActiveIndex(tempIndexArr);
     };
 
+    function handleLogDataViewClose(): void {
+        setView(false);
+        setCurrentLog(null);
+    }
+
+    const handleLogDataView = (logObject: InterfaceLogEntry) => {
```

Comment:
- Please write doc comments for all the newly introduced functions.

- Suggesting a more descriptive name
```suggestion
    const handleLogDataViewOpen = (logObject: InterfaceLogEntry) => {
```